arch/xtensa/src/common/xtensa_initialize.c

```diff
@@ -150,7 +150,7 @@ void up_initialize(void)
 
   /* Initialize the internal heap */
 
-#ifdef CONFIG_XTENSA_USE_SEPERATE_IMEM
+#ifdef CONFIG_XTENSA_USE_SEPARATE_IMEM
```

Comment:
I'm pretty sure I fixed these typos in a previous PR, why are they here?

Reply:
It looks like I either missed some or didn't commit some. Either case I'm confused how it happened.  Anyway, thanks for fixing.